gitVersionConfig.yaml

```diff
@@ -0,0 +1,2 @@
+assembly-versioning-scheme: MajorMinor
+mode: ContinuousDeployment
```

Comment:
Seems like ContinuousDeployment is correct, couldn't get it working locally though :(